clairmeta/utils/file.py

```diff
@@ -87,16 +87,24 @@ def temporary_dir():
         shutil.rmtree(dirpath)
 
 
-def console_progress_bar(file_path, file_progress, total_progress, dcp_size, elapsed, done):
+def console_progress_bar(
+    file_path,
+    file_progress,
+    file_processed,
+    elapsed,
```

Comment:
As this is not a class and cannot store any data and only works on what you pass it..  It's currently not possible to figure out ETA as `elapsed` is only for the current file being read.   What's your recommendation to bring in a `total_elapsed` as so you can calculate a ETA for the command to execute?

Reply:
See my latest commit to have a class based implementation and a naive ETA estimation (as in a mean speed average based on last N seconds would probably be recommend).

Reply:
As for the callback rate, good catch, I saw slowdown from around 750MB/s to 450MB/s with the `-progress` tag.

I updated this so that it's called every 200ms for now, should be enough.